### DISCUSSION TITLE
CSS not working for me

### DISCUSSION THREAD

**Discussion by Carlos-005** [community/community#23663] [Other Feature Feedback, Questions, & Ideas]
<p>Hello I’m new to coding and it seems  like my site doesn’t want to upload the CSS folder could someone please help me  not sure what I’m doing wrong. :exploding_head: :man_shrugging:</p>
<p><aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/Carlos-005/My-Site/blob/34f984c312410736268ae3bbc16590d36abdebc9/index.html" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/Carlos-005/My-Site/blob/34f984c312410736268ae3bbc16590d36abdebc9/index.html" target="_blank" rel="noopener nofollow ugc">Carlos-005/My-Site/blob/34f984c312410736268ae3bbc16590d36abdebc9/index.html</a></h4>
<pre><code class="lang-html">&lt;!DOCTYPE html&gt;
&lt;html lang="en" dir="ltr"&gt;

&lt;head&gt;
  &lt;meta charset="utf-8"&gt;
  &lt;title&gt;Carlos Pires&lt;/title&gt;
  &lt;link rel="stylesheet" href="css/styles.css"&gt;
  &lt;link rel="icon" href="favicon.ico"&gt;
  &lt;link href="https://fonts.googleapis.com/css2?family=Dancing+Script:wght@500&amp;family=Montserrat&amp;family=Roboto&amp;display=swap" rel="stylesheet"&gt;
&lt;/head&gt;

&lt;body&gt;
  &lt;div class="top-container"&gt;
    &lt;img class="top-cloud" src="images/cloud.png" alt="cloud-img"&gt;
    &lt;h1&gt;Hello, I'm Carlos.&lt;/h1&gt;
    &lt;p&gt;&lt;b&gt;&lt;span class="pro"&gt;Newbie&lt;/span&gt; Web Designer&lt;b&gt;&lt;/p&gt;
    &lt;img class="bottom-cloud" src="images/cloud.png" alt="cloud-img"&gt;
    &lt;img src="images/mountain.png" alt="mountain-img"&gt;
  &lt;/div&gt;
  &lt;div class="middle-container"&gt;
</code></pre>

  This file has been truncated. <a href="https://github.com/Carlos-005/My-Site/blob/34f984c312410736268ae3bbc16590d36abdebc9/index.html" target="_blank" rel="noopener nofollow ugc">show original</a>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
<br>
<aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/Carlos-005/My-Site/blob/34f984c312410736268ae3bbc16590d36abdebc9/CSS/styles.css" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/Carlos-005/My-Site/blob/34f984c312410736268ae3bbc16590d36abdebc9/CSS/styles.css" target="_blank" rel="noopener nofollow ugc">Carlos-005/My-Site/blob/34f984c312410736268ae3bbc16590d36abdebc9/CSS/styles.css</a></h4>
<pre><code class="lang-css">body {
  color: #40514E;
  margin: 0;
  text-align: center;
  font-family: 'Roboto', sans-serif;

}

h1 {
  font-size: 5rem;
  color: #5768DE;
  line-height: 2;
  margin: 50px auto 0 auto ;
  font-family: 'Dancing Script', cursive;
}

h2 {
  font-family: 'Montserrat', sans-serif;
  font-weight: normal;
  color: #5768DE;
</code></pre>

  This file has been truncated. <a href="https://github.com/Carlos-005/My-Site/blob/34f984c312410736268ae3bbc16590d36abdebc9/CSS/styles.css" target="_blank" rel="noopener nofollow ugc">show original</a>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
</p>
<p>Thanks.</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/carlos-005">@Carlos-005</a>, welcome to the GitHub Support Community!</p>
<p>GitHub Pages is case sensitive. It looks like you’re referencing your CSS at the path <code>css/styles.css</code>, but your CSS file is actually at <code>CSS/styles.css</code>. You’ll need to either rename your CSS directory to lowercase, or update your <code>link</code> tag to point to the uppercase folder instead.</p>

**Comment by Carlos-005** [comment]
<p>Oh wow thanks man, lol cant believe that was it, I’ll know for the future. Thanks a lot for the help :slight_smile:</p>
